### ISSUE TITLE
Finish docs on proper cache invalidation

### ISSUE THREAD

**Issue by n8kim1** [battlecode/galaxy#60]
**tldr** this is set up on the old project. There's two parts: the backend bucket settings (see below screenshots), and the frontend deploy settings (see the script linked below.) **Both these steps need to be moved to the deploy process and terraform-ized.**
Critical because bad caching creates horrible production bugs.

When we deploy a new frontend update, sometimes Google caches the frontend anyways, and so people don't see the updated frontend for a while. This is especially annoying when we have to fix breaking bugs; push out a tournament update ASAP; etc
A couple things: our website code itself needs to have `Cache-Control: no-cache` in the right place. Also, Google Cloud needs to invalidate its own internal cache as necessary (see Jerry's comment)

(Testing this will be hard; we'll just have to see if it ultimately works in prod)
